src/main/java/com/bookshop/dto/CreateBookRequestDto.java

```diff
@@ -1,12 +1,25 @@
 package com.bookshop.dto;
 
+import com.bookshop.validation.Isbn;
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
 import java.math.BigDecimal;
 
 public record CreateBookRequestDto(
+        @NotNull
+        @Size(max = 255)
         String title,
+        @NotNull
+        @Size(max = 255)
         String author,
+        @Isbn
         String isbn,
+        @NotNull
+        @Min(0)
         BigDecimal price,
+        @Size(max = 255)
```

Comment:
Is it really necessary for description and cover image being not null? @hy we can't add book without description and image?

Reply:
These values are not marked as @NotNull, their length cannot exceed 255 characters, as the corresponding database column is defined as varchar(255), and values longer than that will not be saved.

Reply:
Oh, sorry, that's my bad. To tell the truth, it was 2 am when I was reviewing your code 😂😂